docs/index.rst

```diff
@@ -191,6 +190,40 @@ Symbols
 *******
 Symbols are used in a :ref:`Dialogue <concepts-dialogues>` data structure as programmatically-usable strings. Their main use is for identifying and referencing sequences, blocks and block types.
 
+
+Permissions
+~~~~~~~~~~~
+A user's actions are limited by the permissions they have been granted. Users can be granted the following permissions:
+
+``admin``
+*********
+Grants create, archive, read and write access for all projects and dialogues, and publish access for all dialogues.
+
+``projects:create``
+*******************
+Grants access to create new projects. Users with this permission obtain ``project:admin`` access for the projects they create.
+
+``project:admin``
+*****************
+Grants create, archive, read, write and publish access for a given project's dialogues.
+
+``project:dialogues:read``
+**************************
+Grants read access for a given project's dialogues.
+
+``project:dialogues:write``
+***************************
+Grants write access for a given project's dialogues.
+
+``dialogue:read``
+*****************
+Grants read access for a given dialogue.
+
+``dialogue:write``
+******************
+Grants write access for a given dialogue.
```

Comment:
We could obviously support more permission levels than these: for example, we could include a permission to only create dialogues for a project, or to be able to archive dialogues for a project. I tried to think of which groups of permissions were the most sensible:
  - Being able to create a dialogue in a project but not having read or write access to other dialogues in a project sounds like it could get complex. One thing we would need to handle would be granting read, write and archive access for dialogues that users has created permission (should be able to publish for that dialogue too?). I'm not such complex permissions.
  - If we had a separate publish permission, we would probably need to grant users with that permission read and write access (otherwise, they wouldn't be able to undo their publishes or revert to previous publishes if their newly published dialogue breaks). I'm also not sure if there are cases where one can read, write, and publish for a dialogue, but not publish for other dialogues in the same project (this would get even more complicated once we support delegating from one dialogue to another).

I vote we try stick to simpler permission like the ones above, and add more finer grained permissions as the use cases appear.